db.cpp

```diff
@@ -210,7 +210,12 @@ bool TelegramRecorder::writeMessageToDB(std::shared_ptr<td_api::message>& messag
   statement += (fileOriginID == "" ? "NULL" : "'" + fileOriginID + "'") + ",";
   statement += std::to_string(message->chat_id_) + ",";
   statement += std::to_string(senderID) + ",";
-  statement += (message->reply_to_ ? ("'" + std::to_string(message->reply_to_)  + "'") : "NULL") + ",";
+  std::string reply_to = "";
```

Comment:
Yeah this is actually a much better idea, this way we can still join on message id to know who this is from, as well as better reconstruct the convo.